### ISSUE TITLE
Torchvision.utils make_grid does not normalize

### ISSUE THREAD

**Issue by evcu** [pytorch/vision#404]
[`discuss.thread`](https://discuss.pytorch.org/t/torchvision-utils-make-grid-does-not-normalize/11885
)
Am I missing something? When I provide a single image it doesn't normalize 
```
import torch
import torchvision
from torchvision.utils import make_grid

print(torchvision.__file__) ##.../anaconda3/lib/python3.6/site-packages/torchvision-0.2.0-
print(torch.__version__) ## 0.3.0.post4

a=(torch.rand(4,5)-0.5).unsqueeze(0)
print(a.min()) ##-0.4951971769332886
gr = make_grid(a,normalize=True, scale_each=True,nrow=4,padding=2)
print(gr.min()) ##-0.4951971769332886
```

**Comment by alykhantejani** [comment]
Hi @evcu,

This has been fixed in master (see [here](https://github.com/pytorch/vision/pull/360)) but there has not been a release since
